DXMainClient/DXGUI/Multiplayer/GameInformationPanel.cs

```diff
@@ -126,9 +176,12 @@ public void SetInfo(GenericHostedGame game)
             lblHost.Text = "Host:".L10N("Client:Main:GameInfoHost") + " " + Renderer.GetSafeString(game.HostName, lblHost.FontIndex);
             lblHost.Visible = true;
 
-            lblPing.Text = game.Ping > 0 ? "Ping:".L10N("Client:Main:GameInfoPing") + " " + game.Ping.ToString() + " ms" : "Ping: Unknown".L10N("Client:Main:GameInfoPingUnknown");
+            lblPing.Text = "Ping:".L10N("Client: Main: GameInfoPing");
```

Comment:
The L10N key name does not come with spaces after `:`

Reply:
Fixed